dist/Data-Dumper/Dumper.pm

```diff
@@ -9,9 +9,12 @@
 
 package Data::Dumper;
 
+use strict;
+use warnings;
+
 BEGIN {
-    $VERSION = '2.175'; # Don't forget to set version and release
-}               # date in POD below!
+    our $VERSION = '2.176'; # Don't forget to set version and release
```

Comment:
I still believe this is weird to define these variables in a `BEGIN` block... they should be defined before and set inside the block..

Reply:
```perl
use strict;
use warnings;
use v5.30;
BEGIN {
    our $X = 42;
}
say $X;
```
```
> perl t.pl
Variable "$X" is not imported at t.pl line 10.
Global symbol "$X" requires explicit package name (did you forget to declare "my $X"?) at t.pl line 10.
Execution of t.pl aborted due to compilation errors.
```

Reply:
It is not weird at all. The variable is not needed after the BEGIN block.